website/content/docs/start-with-a-template.md

```diff
@@ -55,6 +55,13 @@ You can add Netlify CMS [to an existing site](/docs/add-to-your-site/), but the
         <h4>Nuxt.js Boilerplate</h4>
         <p><a href="https://app.netlify.com/start/deploy?repository=https://github.com/Knogobert/ntn-boilerplate&amp;stack=cms"><img src="https://www.netlify.com/img/deploy/button.svg" alt="Deploy to Netlify" /></a></p>
     </div>
+    <div style="flex-basis: 33%">
+        <div style="padding: 0 30%; height: 100px; display: flex; justify-content: center;">
+            <img style="display: flex" src="/img/metalsmith.png"/>
```

Comment:
@wernerglinka do you mind adding the logo under `website/static/img`?

The image is broken without it, see https://deploy-preview-6324--netlify-cms-www.netlify.app/docs/start-with-a-template/

<img width="443" alt="image" src="https://user-images.githubusercontent.com/26760571/160104764-83ca0019-1d43-42c7-bf42-998f3f7a7850.png">

Reply:
![metalsmith](https://user-images.githubusercontent.com/346112/160135884-5cc12a21-90ef-417f-afc3-1bec0fa62f02.svg)